hydrolib/core/dflowfm/mdu/models.py

```diff
@@ -561,6 +570,14 @@ class Comments(INIBasedModel.Comments):
         dtinit: Optional[str] = Field(
             "Initial timestep in seconds [s].", alias="dtInit"
         )
+        autotimestepnostruct: Optional[str] = Field(
+            "Exclude structure links (and neighbours) from time step limitation (0 = no, 1 = yes).",
```

Comment:
These two comments do not seem in line with the comments stated in #489, are they correct?

[time]
AutoTimestepNoStruct = 0 **# or 0 = no, 1 = yes (Exclude negative qin terms from time step limitation). Default: 0**.
AutoTimestepNoQout = 1 **# 0 = no, 1 = yes (Exclude structure links (and neighbours) from time step limitation). Default: 1.**

Reply:
Incorrectly state in https://github.com/Deltares/HYDROLIB-core/issues/489, at that time, adjusted in lateron, implementation correct